test/controllers/api/v0/annotations_controller_test.rb

```diff
@@ -166,4 +166,43 @@ class Api::V0::AnnotationsControllerTest < ActionDispatch::IntegrationTest
       'Change#significance was updated by annotation'
     )
   end
+
+  test 'can list annotations' do
+    page = pages(:home_page)
+    change_id = page.versions[0].uuid
+
+    get(api_v0_page_change_annotations_path(page, "..#{change_id}"))
+
+    assert_response :success
+    body_json = JSON.parse @response.body
+    assert(body_json.key?('links'), 'Response should have a "links" property')
+    assert(body_json.key?('data'), 'Response should have a "data" property')
+    assert(body_json.key?('metadata'), 'Response should have a "metadata" property')
+    assert(body_json['data'].is_a?(Array), 'Data should be an array')
+  end
+
+  test 'metadata property' do
+    page = pages(:home_page)
+    change_id = page.versions[0].uuid
+
+    annotation = { 'test_key' => 'test_value' }
+    sign_in users(:alice)
+    post(
+      api_v0_page_change_annotations_path(page, "..#{change_id}"),
+      as: :json,
+      params: annotation
+    )
+    assert_response :success
+
+    get(api_v0_page_change_annotations_path(page, "..#{change_id}"))
+
+    assert_response :success
+    body_json = JSON.parse @response.body
+
+    assert_equal(
+      Change.find_by_api_id("..#{change_id}").annotations.count,
+      body_json['metadata']['total_results'],
```

Comment:
This isn’t really adequate—if there’s only one annotation, all the annotations will always be in the response. It might be better to post two annotations, then use the `chunk_size` parameter in the request to limit it to only one result (i.e. `get(api_v0_page_change_annotations_path(page, "..#{change_id"}, params: {chunk_size: 1}))`).

Most of the other tests should probably do the same with `chunk_size`.